api/client/stack/common.go

```diff
@@ -12,7 +12,9 @@ import (
 )
 
 const (
-	labelNamespace = "com.docker.stack.namespace"
+	// TODO: The `experimental` should be removed once the
+	// stack feature is out of the experimental status
+	labelNamespace = "com.docker.experimental.stack.namespace"
```

Comment:
If we make this change it's going to be harder for users to transition off of experimental. I think we should keep the old label.